src/redux/books/books.reducer.js

```diff
@@ -0,0 +1,31 @@
+import booksActionType from './books.type';
+
+const INITIAL_STATE = {
+  loading: true,
+  books: [],
+  error: false,
+};
+const booksReducer = (state = INITIAL_STATE, action) => {
+  switch (action) {
```

Comment:
- [ ] The switch argument should be `action.type` because we're checking the case that matches the action type. Kindly change line 9 to `switch (action.type) {`